### ISSUE TITLE
Add more discussion of inclusivity

### ISSUE THREAD

**Issue by kelle** [astropy/astropy-v2.0-paper#137]
> Given the title of this paper, I expected to see more discussion by the authors about the inclusivity of their community. An uber-skeptical reader could make the assessment that the Astropy community has a code of conduct therefore believes it's inclusive. I suspect that the community is doing much more than this though... So, has any attempt been made by the authors to quantify the inclusivity of the Astropy community? There's a growing literature in the quantitative social sciences studying open source communities that the authors might want to look into: e.g. the work of Gousios http://www.gousios.gr/publications.html and Vasilescu https://cmustrudel.github.io/publications/ come to mind. Some discussion or attempt to quantify the inclusivity is recommended.

I have two responses to this:
- We can totally write more about our inclusive practices and give some numbers. This is 100% reasonable even though I'm not exactly sure where to get numbers from (mailing lists?, Python in Astronomy, FB group?) and which axes of inclusivity to focus on (gender, ethnicity, country of origin, current country, past IDL users, etc.). Let me know if the any of the primary authors would like me to help out with this.
- Unless we have social scientists knocking on our door wanting to do this study, I am very opposed to us studying ourselves. This activity is a serious undertaking and one which should be done by a professional, not by Astronomers. Also, I think any effort to study the demographics/inclusivity, should be done across all NumFocus projects, and not *just* Astropy. It would be much more powerful to get cross-project comparisons to empower us to suss-out quantifiably supported "best practices".

**Comment by rgommers** [comment]
> Unless we have social scientists knocking on our door wanting to do this study, I am very opposed to us studying ourselves. This activity is a serious undertaking and one which should be done by a professional, not by Astronomers. Also, I think any effort to study the demographics/inclusivity, should be done across all NumFocus projects, and not just Astropy. It would be much more powerful to get cross-project comparisons to empower us to suss-out quantifiably supported "best practices".

This may be of interest to or in scope for the NumFOCUS Diversity & Inclusion in Scientific Computing (DISC) program. DISC is led by Gina, she can hopefully comment. ping @Dr-G

**Comment by Dr-G** [comment]
Thanks for tagging me, @rgommers. @kelle I'd be happy to connect with Astropy about your needs regarding data collection and potentially connecting with social scientists to conduct research on the community. As the DISC Program continues to grow, it will be nice to have the projects come to us to articulate their needs. Let me know.

**Comment by kelle** [comment]
in the paper, I think we should mention the coordination/involvement with NumFocus and DISC, we're working on getting better numbers, and could potentially work with social scientists in the future. 

I should mention here that I was part of a group who started to put together a NumFocus-wide demographics survey at the DISC Unconference in NYC in Oct. I'm not on top of the status of that project, but just want to be explicit about the Astropy Project's involvement with that program.

**Comment by hamogu** [comment]
I think part of the problem is that "inclusive" means different things to different people as @kelle pointed out above. A full study of this is "beyond the scope of the current article" in my opinion. I've always understood astropy's inclusivity as "anyone who wants", which essentially means "the astronomical community at large". By many measures of "inclusive" the astronomical community already a very biased sample. I'm not aware of astropy reaching out specifically to e.g. underrepresented ethnic groups. 
If we understand "inclusive" as "inclusive of the astronomical community", then we can make a point about strong participation from all academic levels - there is no faculty-only steering committee (e.g. @eteq was a grad student in the beginning) and no review panels that are filled with mid-to-late career stage people. 
We can look e.g. at the people with commit rights to the main repository and determine career stages when they first got the permissions. That is fast to do, because is is only a few dozen people that we the authors no well and that are generally very responsive by email.
I think that is actually a defining characteristic that sets astropy apart from other astronomical projects. Anything beyond that is, in my opinion, beyond the scope of the article; we might also consider "tuning down" or mentions of inclusiveness a little since we cannot actually back it up with numbers from a professional social science study.

**Comment by eblur** [comment]
Given the title of the paper, I agree with the referee that this issue is **major**.  It needs at least a paragraph defining what we mean by inclusion.

Perhaps changing the title to "The Astropy Project: Status of the v2.0 core and efforts towards building an inclusive, open-science project" will be closer to reality.

I agree that putting numbers to demographics such as gender, ethnicity, country of origin are beyond the scope of the paper and dodgy if we attempt to study ourselves. A paragraph could focus on the planned collaboration with NumFOCUS DISC.

Giving a few concrete guesses like @hamogu suggests also sounds good. We could say that underlying demographics are unknown, but career stage might be taken as an indicator.

.... editing as I skim the latest draft ...

**Comment by eblur** [comment]
Just because everyone can -- in theory -- contribute code, doesn't mean that everyone who can will contribute. That's the difficulty with inclusivity as a topic. The barriers are invisible to those who are already included.

I think the paper needs a subsection on Inclusivity -- outlining the Code of Conduct, target demographic axes, and future efforts -- to warrant using "inclusive" in the title.

I know this is more work than expected. I can volunteer time to help write it, if needed.

**Comment by hamogu** [comment]
@eblur Great! Would you be willing to do a PR with a draft for such a section? It's easier to discuss an actual piece of text rather than the abstract concept here, I think.

**Comment by dhuppenkothen** [comment]
Would it be appropriate to also talk about efforts to recruit a more diverse contributor base via the Python in Astronomy workshops? Diversity and inclusivity have always been a part of PyAstro efforts, so that might be at least worth mentioning?

For any sort of survey that contains demographic information (and I think asking people about their career stage counts), you might need to think about IRB approval. That's worth keeping in mind.

**Comment by hamogu** [comment]
The good think about "career stage of people with commit rights" is something we don't have to ask for - You or I or @eblur know most of them well enough that we can just count.

**Comment by bsipocz** [comment]
I like @hamogu's idea, the people with commit rights is a big enough group, yet a manageable size to even follow-up individually. Also, some biases from the conclusions might have been removed, as everyone currently having the rights had to significantly contribute before getting it.

Also, if we talk about actual contributors, I would argue that we got more, and also a more diverse group in terms of country of origin, gender, student career stage etc via gsoc than pyastro.
